bundling/bundle-web.ts

```diff
@@ -7,14 +7,9 @@ const [release, source = `https://deno.land/x/grammy@${release}/mod.ts`] =
 if (!release) throw new Error("No release specified!");
 
 // Rewrite imports from .deno.ts to .web.ts
-const cache = createCache();
-const load = (specifier: string) => {
-    if (specifier.endsWith(".deno.ts")) {
-        const baseLength = specifier.length - ".deno.ts".length;
-        specifier = specifier.substring(0, baseLength) + ".web.ts";
-    }
-    return cache.load(specifier);
-};
+const cache = createCache({ root: "deno_cache/" });
+const load = (specifier: string) =>
+    cache.load(specifier.replace(/\.deno\.ts$/, ".web.ts"));
```

Comment:
Could you revert to using explicit string operations rather than regex? I inherently don't trust ~~regex~~ humans brains to read regex correctly